Index: app/src/main/java/ru/tubi/project/free/General_information.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/ru/tubi/project/free/General_information.java	(revision fb14440e36d4f0da0ebdf10e8e79523068da6376)
+++ app/src/main/java/ru/tubi/project/free/General_information.java	(date 1659631669764)
@@ -13,7 +13,6 @@
     залить релиз в Google play
 
     сверить БД таблицы столбцы все в наличии
-
     в PHP сменить имя БД на h102582557_tubi
     на сервере в api_tubi удалить все файлы php, а папки с фото не трогать(image, preview_image)
     залить обновление в api_tubi
